src/open_prime_hunters_rando/asm/patches/missile_launcher.s

```diff
@@ -0,0 +1,21 @@
+b       =0x022E8000     // jump to unused space
+
+/*
+Register values
+R0: 21 (id value of Affinity Weapon)
+R1: #0
+R2: value of the ammo cap
+R8: ???
+R12: more ammo caps?
+*/
+
+// ammo caps
+add     r12, r8, #0x100
+ldrh    r2, [r12,#0x52]
+ldr     r0, [sp,#0x38]  // state
+add     r2, r2, #0x32   // ammo given when picked up (can be modified)
+mov     r1, #0x37       // string message id
+mov     lr, 10          // game reads ammo as value * 10 so prep for multiplication
```

Comment:
does this not have to be `mov lr, #10`? i thought constant/immedaite numbers have to have a pound character in front of them